header.php

```diff
@@ -126,10 +128,12 @@
               }
             ?>
             <?php 
-              if (!empty($vip_login_url)) { 
-                echo '<a href="' . $vip_login_url . '" class="border-underline margin-bottom-micro">';
-                echo __('[:en]VIP Login[:es]Sección VIP');
-                echo '</a>';
+              if (!empty($vip_login_url) && !empty($show_vip_login)) { 
+                if ($show_vip_login == 'on') {
```

Comment:
These conditions could be a single condition. 
`if (!empty($vip_login_url) && $show_vip_login == 'on')`
no?